src/insns/prefetch.w.adoc

```diff
@@ -49,6 +44,17 @@ likely to be accessed by a data write (i.e. store) in the near future.  The
 encoding is only valid if imm[4:0]=0. The authorising capability for this
 operation is <<ddc>>.
 
+In either mode, PREFETCH.W does not perform a memory access
+if one or more of the following conditions of the authorising capability are met:
+
+* The tag is not set
```

Comment:
Not necessary to fix this before merging, but maybe worth factoring this out into a helper file since the only difference is the permission.

Reply:
Done